io.github.francivaldo4334.Bitsafe_cli.yaml

```diff
@@ -0,0 +1,62 @@
+            sha256: "e6bf8de6c36ed96c86ea3b6e1d5273c53f46ef518a062464cd7ef5dd2cf92e38"
+          - type: file
+            url: "https://files.pythonhosted.org/packages/3e/b9/3766cc361d93edb2ce81e2e1f87dd98f314d7d513877a342d31b30741680/pypng-0.20220715.0-py3-none-any.whl"
+            sha256: "4a43e969b8f5aaafb2a415536c1a8ec7e341cd6a3f957fd5b5f32a4cfeed902c"
+          - type: file
+            url: "https://files.pythonhosted.org/packages/24/79/aaf0c1c7214f2632badb2771d770b1500d3d7cbdf2590ae62e721ec50584/qrcode-7.4.2-py3-none-any.whl"
+            sha256: "581dca7a029bcb2deef5d01068e39093e80ef00b4a61098a2182eac59d01643a"
+          - type: file
+            url: "https://files.pythonhosted.org/packages/b7/f4/6a90020cd2d93349b442bfcb657d0dc91eee65491600b2cb1d388bc98e6b/typing_extensions-4.9.0-py3-none-any.whl"
+            sha256: "af72aea155e91adfc61c3ae9e0e342dbc0cba726d6cba4b6c72c1f34e47291cd"
+
+      - name: python3-requests
+        buildsystem: simple
+        build-commands:
+          - "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"requests~=2.31.0\" --no-build-isolation"
+        sources:
+          - type: file
+            url: "https://files.pythonhosted.org/packages/64/62/428ef076be88fa93716b576e4a01f919d25968913e817077a386fcbe4f42/certifi-2023.11.17-py3-none-any.whl"
+            sha256: "e036ab49d5b79556f99cfc2d9320b34cfbe5be05c5871b51de9329f0603b0474"
+          - type: file
+            url: "https://files.pythonhosted.org/packages/63/09/c1bc53dab74b1816a00d8d030de5bf98f724c52c1635e07681d312f20be8/charset-normalizer-3.3.2.tar.gz"
+            sha256: "f30c3cb33b24454a82faecaf01b19c18562b1e89558fb6c56de4d9118a032fd5"
+          - type: file
+            url: "https://files.pythonhosted.org/packages/c2/e7/a82b05cf63a603df6e68d59ae6a68bf5064484a0718ea5033660af4b54a9/idna-3.6-py3-none-any.whl"
+            sha256: "c05567e9c24a6b9faaa835c4821bad0590fbb9d5779e7caa6e1cc4978e7eb24f"
+          - type: file
+            url: "https://files.pythonhosted.org/packages/70/8e/0e2d847013cb52cd35b38c009bb167a1a26b2ce6cd6965bf26b47bc0bf44/requests-2.31.0-py3-none-any.whl"
+            sha256: "58cd2187c01e70e6e26505bca751777aa9f2ee0b7f4300988b709f44e013003f"
+          - type: file
+            url: "https://files.pythonhosted.org/packages/96/94/c31f58c7a7f470d5665935262ebd7455c7e4c7782eb525658d3dbf4b9403/urllib3-2.1.0-py3-none-any.whl"
+            sha256: "55901e917a5896a349ff771be919f8bd99aff50b79fe58fec595eb37bbc56bb3"
+
+  - name: bitsafe
+    buildsystem: simple
+    build-commands:
+      - install -D bsafe /app/bin/bsafe
+      - install -Dm644 io.github.francivaldo4334.Bitsafe_cli.desktop /app/share/applications/io.github.francivaldo4334.Bitsafe_cli.desktop
+      - install -Dm644 io.github.francivaldo4334.Bitsafe_cli.svg /app/share/icons/hicolor/scalable/apps/io.github.francivaldo4334.Bitsafe_cli.svg
```

Comment:
https://github.com/francivaldo4334/Bitsafe_cli/blob/main/Dm644%20io.github.francivaldo4334.Bitsafe_cli.svg icon name has a stray Dm644 in front